confgenerator/filter/filter.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filter
```

Comment:
Add a `//go:generate` comment to regenerate the lexer code automatically?

Reply:
Great idea, added! Didn't know that was a thing, neat.

Reply:
Whoops, I assumed it had to be the first line for some reason, but that's not the case. Will fix.